src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1508,12 +1516,162 @@ void invalidMultipleHandles() {
+
+                markImportance(makeHypothesis(model, getUri("hypothesis-1"),
+                        ImmutableSet.of(event, relation), system), 100);
+
+            }
+
+            @Test
+            void invalidEvent() {
+                //invalid event cluster, no importance value
+                makeClusterWithPrototype(model, getClusterUri(), event, system);
+                markImportance(makeClusterWithPrototype(model, getClusterUri(), relation, system), 99);
+                testInvalid("NISTHypothesis.invalid: Each event or relation (cluster) in the hypothesis must " +
+                        "have exactly one importance value");
+            }
+
+            @Test
+            void invalidRelation() {
+                //invalid relation cluster, no importance value
+                markImportance(makeClusterWithPrototype(model, getClusterUri(), event, system), 88);
+                makeClusterWithPrototype(model, getClusterUri(), relation, system);
+                testInvalid("NISTHypothesis.invalid: Each event or relation (cluster) in the hypothesis must " +
+                        "have exactly one importance value");
+            }
+
+            @Test
+            void valid() {
+                markImportance(makeClusterWithPrototype(model, getClusterUri(), event, system), 88);
+                markImportance(makeClusterWithPrototype(model, getClusterUri(), relation, system), 99);
+                testValid("NISTHypothesis.valid: Each event or relation (cluster) in the hypothesis must " +
+                        "have exactly one importance value");
+            }
+        }
+
+        // Each edge KE in the hypothesis graph must have exactly one edge importance value
+        @Nested
+        class HypothesisEdgeImportanceValue {
+
+            private final String documentEventUri = getUri("ldc:event-1");
+            private final String relationUri = getUri("ldc:relation-1");
+            Resource event;
+            Resource relation;
+
+            @BeforeEach
+            void setup() {
+
+                event = makeEvent(model, documentEventUri, system);
+                markJustification(markType(model, getAssertionUri(), event,
+                        SeedlingOntology.Personnel_Elect, system, 1.0), justification);
+
+                relation = makeRelation(model, relationUri, system);
+                markJustification(markType(model, getAssertionUri(), relation,
+                        SeedlingOntology.GeneralAffiliation_APORA, system, 1.0), justification);
+
+                markImportance(makeClusterWithPrototype(model, getClusterUri(), event, system), 88);
+                markImportance(makeClusterWithPrototype(model, getClusterUri(), relation, system), 88);
+            }
+
+            @Test
+            void invalidEventEdge() {
+
+                //invalid event argument, needs importance value
+                markAsArgument(model, event, SeedlingOntology.Personnel_Elect_Elect,
```

Comment:
You might want to add a URI for the argument assertions so that your invalid messages will be easier to read. This applies to relations as well

Reply:
Updated.